citrushack/cutiehack2022
# create userbar

the only changes really made is the modal styling and adding the github provider login method
test

- make sure both google and github logins work (email doesn't work at the moment, it isn't setup correctly, this is a separate issue)

things done in this issue
- add basic themes
- add dark mode
- add modal to allow users to sign in
- add userbar for users to sign in/out change color theme

the colors are obviously set to change later once ui/ux & marketing team pulls through
